lua/telescope/pickers.lua

```diff
@@ -787,7 +790,7 @@ function Picker:refresh(finder, opts)
   if finder then
     self.finder:close()
     self.finder = finder
-    self._multi = MultiSelect:new()
+    self._multi = vim.F.if_nil(opts.multi, MultiSelect:new())
```

Comment:
Probably a good idea to put this in the docs bit above like `opts.reset_prompt`. Can also explain what sort of object to be passed here.

Reply:
Good idea :) I streamlined the docs. Side note: probably eventually good to push for broader docs again (eg get #927 merged).